src/components/skills-quiz/__mocks__/react-instantsearch-dom.jsx

```diff
@@ -29,6 +29,11 @@ const fakeHits = [
   },
 ];
 
+// This allows you to override the built-in hits object
+const setFakeHits = hits => {
```

Comment:
[consideration] Is there risk of unintentionally changing the default `fakeHits` between tests?

Reply:
I've added reset logic between tests that can be called to have the mocked value reset to the default mocked value after each test 👍🏽 . This should prevent any data pollution that make occur between tests.